### PR TITLE
Inserter: Fix inserter font style and adding a position prop

### PR DESCRIPTION
**Pull request by youknowriad**
In this PR, I'm fixing the style of the inserter (fonts) and I'm adding a "position" prop to position the inserter menu at the bottom or at the top of the button.

The inserter menu shown on the header is displayed under the button and the one at the end of the editor shows the menu above the button.

### CODE REVIEW

**Comment by jasmussen** [comment]
Pushed a few further tweaks. It will need more love in the future, but I think this is good for now. 👍

File: editor/components/inserter/menu.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function InserterMenu() {
 	const categories = wp.blocks.getCategories();
 
 	return (
-		<div className="editor-inserter__menu">
+		<div className={ `editor-inserter__menu is-${ position }` }>
```
Comment (aduth) [inline editor/components/inserter/menu.js]: Should we have used `classnames` here? I guess it's a simple enough concatenation.

File: editor/components/inserter/style.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@
 	background: none;
 	border: none;
 	padding: 0;
+	font-family: $default-font;
+	font-size: $default-font-size;
+	line-height: $default-line-height;
+  z-index: 1;
```
Comment (aduth) [inline editor/components/inserter/style.scss]: Mixed spacing / tabs.

File: editor/components/inserter/style.scss [hunk 3]
```diff
@@ -31,8 +35,61 @@
...
+			}
+		}
+	}
+
+	&.is-bottom {
+		top: 42px;
+
+		.editor-inserter__arrow {
+			top: -10px;
+			left: 50%;
```
Comment (aduth) [inline editor/components/inserter/style.scss]: Seems a decent amount of these styles could have been left in the common unscoped `.editor-inserter__arrow` selector.

`left`
`margin-left`
`border-left-color`
`border-right-color`
`&:before content`
`&:before position`
`&:before position`
`&:before left`
`&:before border-left-color`
`&:before border-right-color`

I guess a point could be made to having related styles colocated despite repetition.

File: editor/components/inserter/menu.js (hunk 1 above)
Comment (youknowriad) [inline editor/components/inserter/menu.js]: Yeah that was the first thing I did but I reverted for simplicity.

File: editor/components/inserter/style.scss (hunk 2 above)
Comment (youknowriad) [inline editor/components/inserter/style.scss]: oops

**Pull request merged**